pymediainfo/__init__.py

```diff
@@ -219,7 +219,7 @@ def can_parse(cls, library_file=None):
     @classmethod
     def parse(cls, filename, library_file=None, cover_data=False,
             encoding_errors="strict", parse_speed=0.5, text=False,
-            full=True, legacy_stream_display=False):
+            full=True, legacy_stream_display=False, instance_options=None):
```

Comment:
I'm not a fan of the name, perhaps something like `custom_options`, `additional_options`, `mediainfo_options` would be more self-explanatory, what do you think?

Please set the default to `{}` to avoid the additional `if`.

Reply:
I removed `if`, but I would like to avoid adding mutable as a default  function argument. I understand that we don't change `mediainfo_options` inside of `parse` function, but still I think it is a good practice to avoid it.

Reply:
I wasn't aware of the mutable argument gotcha, my bad :) I think I preferred the first version with the `if` then, can you bring it back?
Sorry for the trouble.